backend/oai.py

```diff
@@ -111,6 +114,43 @@ def ingest(self, from_date=None, metadataPrefix='oai_dc',
         records = self.client.listRecords(**args)
         self.process_records(records, metadataPrefix)
 
+    def load_base_dump(self, directory_path):
```

Comment:
We could probably expose this function in a `manage.py` command, similar to https://github.com/dissemin/dissemin/pull/754/files#diff-3e67c0e4d4dd1d92d9ae8a339953cbdbR1.

I can make it and complete the doc introduced as part of #754 with BASE counterpart in #754.

Reply:
Great, that would be very useful I think!

Reply:
Cool, I moved this to #754.